mxcube3/__init__.py

```diff
@@ -83,10 +83,11 @@ def complete_initialization(app):
         app.collect = app.beamline.getObjectByRole("collect")
         app.diffractometer = app.beamline.getObjectByRole("diffractometer")
 	try:
-	    app.diffractometer.centring_motors_list
+	    if getattr(app.diffractometer, 'centring_motors_list') == None:
```

Comment:
I also have my brain on holidays ;) Of course with the broken Mockup we have it always
return `None`.
But this code is not good neither (sorry): when used with a normal diffractometer object, it would never be equal to `None` and it would never do `app.diffractometer.centring_motors_list = app.diffractometer.getPositions().keys()`.
So, I suggest to use the third argument to `getattr`, to specify `None` by default.

Code would be:

```
    if getattr(app.diffractometer, 'centring_motors_list', None) is None:
        app.diffractometer.centring_motors_list = app.diffractometer.getPositions().keys()
```

There is no need for `try...except` anymore, then.

Could you please try it and change your PR accordingly if it works ?

Thanks in advance !